web/src/App.tsx

```diff
@@ -88,6 +89,7 @@ export default function App(): ReactElement {
               </Suspense>
               <Suspense>
                 <OnboardingModal />
+                <AnnouncementModal />
```

Comment:
I was a bit unsure about where to add the `AnnouncementModal` here

Reply:
Looks like the correct location but it should probably be in it's own suspense boundary and possible dynamically imported, especially if the SVG is imported directly in the component.

Reply:
I tried to put it in its own suspense but the e2e test failed. I think it is because the announcementmodal should only be shown if the onboardingmodal has not been shown in the current session, so it needs to wait fo the onboarding modal to load before it can be loaded correctly.